sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreErrors.scala

```diff
@@ -255,6 +255,22 @@ class StateStoreValueSchemaNotCompatible(
       "storedValueSchema" -> storedValueSchema,
       "newValueSchema" -> newValueSchema))
 
+class StateStoreSnapshotFileNotFound(fileToRead: String, clazz: String)
```

Comment:
Can we add wrapper functions similar to https://github.com/apache/spark/pull/46944/files#diff-ca7eb6e6ff6a5638afe83d942081e806e8c592ea0941057a4bf501fc2172824fR147 and use those in the callers ?